src/main/java/uk/gov/gchq/palisade/resource/AbstractLeafResource.java

```diff
@@ -19,6 +19,7 @@
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
+import uk.gov.gchq.palisade.Generated;
 import uk.gov.gchq.palisade.ToStringBuilder;
 
 import java.util.HashMap;
```

Comment:
Use setters for the methods below

Reply:
The AbstractLeafResource methods below need to use setters and not check null etc.
lines 38 - 62